jax/lib/xla_bridge.py

```diff
@@ -202,14 +204,26 @@ def devices(backend=None):
   return get_backend(backend).devices()
 
 
-def local_devices(host_id=None, backend=None):
-  """Returns a list of devices local to a given host (this host by default)."""
+def local_devices(host_id: int = None, backend: str = None):
+  """Like ``devices``, but only returns devices local to a given host.
+
+  If ``host_id`` is ``None``, returns devices local to this host.
```

Comment:
Will this give an error if we pick the host_id and a backend that does not exist on that host?

Reply:
I just tried it. A bad backend raises an error, but a bad host_id currently returns an empty list. I made it raise an error and added a small test.